tilt_devnet/docker_images/Dockerfile.lerna

```diff
@@ -0,0 +1,35 @@
+# The aim for the base image here is to only keep package*.json files while having the
+# same directory structure to be able to have lerna dependencies installed once and cached
+# as long as the package*.json files have not changed. In the future, we can further optimize
+# it by creating slim package files that only contain *dependencies and name keys.
+
+FROM node:18.13.0@sha256:d9061fd0205c20cd47f70bdc879a7a84fb472b822d3ad3158aeef40698d2ce36 as base
+
+WORKDIR /home/node
+COPY --chown=1000:1000 ./ ./
+
+# Remove files that are not json packages
+RUN find . -type f ! -name 'package*.json' -delete
+# Remove directories that are empty now
+RUN find . -type d -empty -delete
```

Comment:
Is this a caching trick?

Reply:
yeah, i didn't want a new folder trigger a new rebuild. the above line also removes all non package*.json files. I explained as a comment above.